tests/upgrade/test_crossgrade.sh

```diff
@@ -116,38 +117,55 @@ installIstioSystemAtVersionHelmTemplate() {
         auth_opts="--set global.mtls.enabled=true --set global.controlPlaneSecurityEnabled=true "
     fi
     release_path="${3}"/install/kubernetes/helm/istio
-    helm init --client-only
-    helm dep update "${release_path}"
+    if [[ "${release_path}" != *"1.0"* ]]; then
```

Comment:
I'd leave this as == 1.1 for one very good reason that you're pointing to 1.1 release below.

Reply:
OK you win, for now. =)

Will probably fix this by adding charts to the release tarball so we don't need to rely on that URL here.